docs/non-normative/groups/system/use-cases.md

```diff
@@ -0,0 +1,77 @@
+# **System Semantic Conventions: General Use Cases**
+
+This document is a collection of the use cases that we want to cover with the System Semantic Conventions. The use cases outlined here inform the working group’s decisions around what instrumentation is considered **required**.  
+Use cases in this document will be stated in a generic way that does not refer to any potentially existing instrumentation in semconv as of writing, such that when we do dig into specific instrumentation, we understand their importance based on our holistic view of expected use cases.
+
+## *Legend*
+
+`General Information` \= The information that should be discoverable either through the entity, metrics, or metric attributes.
+
+`Dashboard` \= The information that should be attainable through metrics to create a comprehensive dashboard.
+
+`Alerts` \= Some examples of common alerts that should be creatable with the available information.
+
+## **Host**
+
+A user should be able to monitor the health of a host, including monitoring resource consumption, unexpected errors due to resource exhaustion or malfunction of core components of a host or fleet of hosts (network stack, memory, CPU…).
+
+### General Information
+
+* Machine name  
+* ID (relevant to its context, could be a cloud provider ID or just base machine ID)  
+* OS information (platform, version, architecture, etc)  
+* Number of CPU cores  
+* Memory Capacity
+
+### Dashboard
+
+* Memory utilization  
+* CPU utilization  
+* Disk utilization  
+* Disk throughput  
+* Network traffic
+
+### Alerts
+
+* VM is up  
+* Memory/CPU/Disk utilization goes above a % threshold  
+* Network activity spikes unexpectedly
+
+## Notes
+
+* The alerts in particular should be capable of being uniformly applied to an entire fleet of hosts  
+* The user may be monitoring a virtualization host i.e. VMWare or Proxmox, and the instrumentation to monitor the health of the root host and the virtual machines it's spawned can be largely the same
+
+## **Process**
+
+A user should be able to monitor the health of an arbitrary process using data provided by the OS.  
+Reasons a user may want this:
+
+1. The process they want to monitor isn't covered by more specific semconv instrumentation such as language runtime metrics, db, http, etc.  
```

Comment:
maybe something along these lines?

```suggestion
1. The process they want to monitor doesn't have in-process runtime-specific instrumentation enabled or is not instrumentable at all such as antivirus or another background process.  
```

i.e. what matters seems to be whether you have in-proc instrumentation and if you can apply it at all.

Reply:
Done in [487af83](https://github.com/open-telemetry/semantic-conventions/pull/1618/commits/487af83f99a9ab8eeeb937e6cfac28638114cd8b)